user-service/routes/history.routes.js

```diff
@@ -11,45 +11,18 @@ module.exports = function (app) {
   });
 
   app.post("/api/hist/save", [authJwt.verifyToken], controller.addHistory);
-  // app.post('/api/user/history', [authJwt.verifyToken], controller.addHistory);
 
   app.post(
     "/api/hist/customsave",
     [authJwt.verifyToken],
     controller.addCustomHistory
   );
-  // app.post('/api/hist/customhistory', [authJwt.verifyToken], controller.addCustomHistory);
 
-  app.get(
-    "/api/hist/get",
-    [authJwt.verifyToken],
-    controller.getAllUniqueQuestions
-  );
-  // app.get('/api/user/history', [authJwt.verifyToken], controller.getAllUniqueQuestions);
-
-  app.get(
-    "/api/hist/get/:difficulty",
-    [authJwt.verifyToken],
-    controller.getAllUniqueQuestionsByDifficulty
-  );
-
-  // app.get(
-  //   '/api/user/history/:difficulty',
-  //   [authJwt.verifyToken],
-  //   controller.getAllUniqueQuestionsByDifficulty
-  // );
   app.get(
     "/api/hist/attempts",
     [authJwt.verifyToken],
     controller.getAttemptedDates
   );
```

Comment:
Should this be one line as well? (same as the ones above)

```suggestion
  app.get("/api/hist/attempts", [authJwt.verifyToken], controller.getAttemptedDates );
```

Reply:
can't bc of auto-formatting when save :(